Stepper: add Getting Started (#7318)
DevExpress/devextreme-documentation: concepts/05 UI Components/Stepper/05 Getting Started with Stepper/05 Create Stepper.md

---
##### jQuery

> [Add DevExtreme to your jQuery application](/concepts/58%20jQuery%20Components/05%20Add%20DevExtreme%20to%20a%20jQuery%20Application/00%20Add%20DevExtreme%20to%20a%20jQuery%20Application.md '/Documentation/Guide/jQuery_Components/Add_DevExtreme_to_a_jQuery_Application/') and use the code below to create a Stepper component with one step. This example utilizes the [items[]](/Documentation/ApiReference/UI_Components/dxStepper/Configuration/items/) array, but you can use [dataSource](/Documentation/ApiReference/UI_Components/dxStepper/Configuration/dataSource/) to define steps as well. If you do not specify at least one step, Stepper will display no data.

Reviewer: ```suggestion
[Add DevExtreme to your jQuery application](/concepts/58%20jQuery%20Components/05%20Add%20DevExtreme%20to%20a%20jQuery%20Application/00%20Add%20DevExtreme%20to%20a%20jQuery%20Application.md '/Documentation/Guide/jQuery_Components/Add_DevExtreme_to_a_jQuery_Application/') and use the code below to create a Stepper component with one step. This example utilizes the [items[]](/Documentation/ApiReference/UI_Components/dxStepper/Configuration/items/) array, but you can use [dataSource](/Documentation/ApiReference/UI_Components/dxStepper/Configuration/dataSource/) to define steps as well. If you do not specify at least one step, Stepper does not display any data.
```

---
##### jQuery
[Add DevExtreme to your jQuery application](/concepts/58%20jQuery%20Components/05%20Add%20DevExtreme%20to%20a%20jQuery%20Application/00%20Add%20DevExtreme%20to%20a%20jQuery%20Application.md '/Documentation/Guide/jQuery_Components/Add_DevExtreme_to_a_jQuery_Application/') and use the code below to create a Stepper component with six steps. This example utilizes the [items[]](/Documentation/ApiReference/UI_Components/dxStepper/Configuration/items/) array, but you can use [dataSource](/Documentation/ApiReference/UI_Components/dxStepper/Configuration/dataSource/) to define steps as well. If you do not specify at least one step, Stepper will display no data.

    <!-- tab: index.js -->
    $(function() {
        $("#stepper").dxStepper({
            items: [

>                 {},

Reviewer 2: Strange format for declaring of items
